notifications/models.py

```diff
@@ -37,7 +37,7 @@ class Notification(models.Model):
                    'about the notification (e.g. how an achievement is '
                    'unlocked or how long a project report is overdue).'))
     image_url = models.CharField(
-        blank=True,
+        blank=False,
```

Comment:
Please clarify why this 'image_url' is set to being a required field (in other words, must be filled in as 'blank' being False (meaning CAN'T be blank))